plugins/obs-ffmpeg/obs-ffmpeg-formats.h

```diff
@@ -62,50 +62,6 @@ obs_to_ffmpeg_video_format(enum video_format format)
 	return AV_PIX_FMT_NONE;
 }
 
-static inline enum video_format
-ffmpeg_to_obs_video_format(enum AVPixelFormat format)
-{
-	switch (format) {
-	case AV_PIX_FMT_YUV444P:
-		return VIDEO_FORMAT_I444;
-	case AV_PIX_FMT_YUV444P12LE:
-		return VIDEO_FORMAT_I412;
-	case AV_PIX_FMT_YUV420P:
-		return VIDEO_FORMAT_I420;
-	case AV_PIX_FMT_NV12:
-		return VIDEO_FORMAT_NV12;
-	case AV_PIX_FMT_YUYV422:
-		return VIDEO_FORMAT_YUY2;
-	case AV_PIX_FMT_UYVY422:
-		return VIDEO_FORMAT_UYVY;
-	case AV_PIX_FMT_RGBA:
-		return VIDEO_FORMAT_RGBA;
-	case AV_PIX_FMT_BGRA:
-		return VIDEO_FORMAT_BGRA;
-	case AV_PIX_FMT_GRAY8:
-		return VIDEO_FORMAT_Y800;
-	case AV_PIX_FMT_BGR24:
-		return VIDEO_FORMAT_BGR3;
-	case AV_PIX_FMT_YUV422P:
-		return VIDEO_FORMAT_I422;
-	case AV_PIX_FMT_YUV422P10LE:
-		return VIDEO_FORMAT_I210;
-	case AV_PIX_FMT_YUVA420P:
-		return VIDEO_FORMAT_I40A;
-	case AV_PIX_FMT_YUVA422P:
-		return VIDEO_FORMAT_I42A;
-	case AV_PIX_FMT_YUVA444P:
-		return VIDEO_FORMAT_YUVA;
-#if LIBAVUTIL_BUILD >= AV_VERSION_INT(56, 31, 100)
```

Comment:
It looks like this function is modified, and then deleted. Just delete the function first without modifying it. This commit should be first in the PR.